cmake/common.cmake

```diff
@@ -21,21 +21,6 @@ macro(_spl_get_absolute_path out in)
     endif()
 endmacro()
 
-macro(spl_checkout_git_submodules)

```

Comment:
Seems to me that here this was remaining stuff forgotten in a last tidy-up-session.